### DISCUSSION TITLE
How to send command to Berry script

### DISCUSSION THREAD

**Discussion by dannyv1101** [arendst/Tasmota#17273] [Q&A]
Hi, how can i send a command from iobroker to the Berry script console via mqtt? I want to control the dac pin of an esp32.

**Comment by barbudor** [comment]
See [how to add command](https://tasmota.github.io/docs/Berry-Cookbook/#compiling-berry) to Tasmota with berry
Then a Berry command behave the same as a native command :  cmnd/topic/commandname

**Reply by sfromis** [reply]
In this thread, I posted a simple example of creating a `DACVoltage` command for that.
https://github.com/arendst/Tasmota/discussions/17035

**Comment by dannyv1101** [comment]
Ok thank you very much,it works fine!
